module/VuFind/sql/migrations/pgsql/9.0/002-add-feedback-table.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TYPE feedback_status AS enum('open','in progress','pending','answered','closed');
+CREATE TABLE feedback (
+id SERIAL,
+user_id int DEFAULT NULL,
+message text,
+form_data json DEFAULT '',
+form_name varchar(255) NOT NULL,
+created timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+updated timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+updated_by int DEFAULT NULL,
+status feedback_status NOT NULL DEFAULT 'open',
+site_url varchar(255) NOT NULL,
+PRIMARY KEY (id),
+KEY user_id (user_id),
```

Comment:
Is there a reason this key exists in the migration but not in the main pgsql.sql file?

Reply:
Good catch, there is no reason, I just missed it somehow...